### PR TITLE
Add discussion around label normalisation.

### PR DESCRIPTION
**Pull request by brian-brazil**
Per discussions at dev meeting after PromCon.

Also add 'service' to list of labels to avoid while I'm here.

@SuperQ

### CODE REVIEW

File: content/docs/instrumenting/writing_exporters.md [hunk 1]
```diff
@@ -153,10 +153,10 @@ advice](/docs/practices/instrumentation/#things-to-watch-out-for) on labels.
 
 Avoid `type` as a label name, it’s too generic and meaningless. You should also
 try where possible to avoid names that are likely to clash with target labels,
-such as `region`, `zone`, `cluster`, `availability_zone`, `az`, `datacenter`,
-`dc`, `owner`, `customer`, `stage`, `environment` and `env` - though if that’s
-what the application calls something it’s best not to cause confusion by
-renaming it.
+such as `region`, `zone`, `service``cluster`, `availability_zone`, `az`,
```
Comment (SuperQ) [inline content/docs/instrumenting/writing_exporters.md]: Missing `, ` after service.

Comment (SuperQ) [reply content/docs/instrumenting/writing_exporters.md]: Wait, no, this is a duplicate of `service` below.

Comment (brian-brazil) [reply content/docs/instrumenting/writing_exporters.md]: Bah, messed up trying to group things together. Fixed.

File: content/docs/instrumenting/writing_exporters.md [hunk 2]
```diff
@@ -206,6 +206,22 @@ If your monitoring exposes a total like this, drop the total. If you have to
 keep it around for some reason (e.g. the total includes things not counted
 individually), use different metric names.
 
+Instrumentation labels should be minimal, every extra label is one more that
+users need to consider when writing their PromQL. Accordingly avoid having
```
Comment (jamtur01) [inline content/docs/instrumenting/writing_exporters.md]: Accordingly,

File: content/docs/instrumenting/writing_exporters.md [hunk 3]
```diff
@@ -206,6 +206,22 @@ If your monitoring exposes a total like this, drop the total. If you have to
...
+instrumentation labels which could be removed without affecting uniqueness of
```
Comment (jamtur01) [inline content/docs/instrumenting/writing_exporters.md]: `the uniqueness of` ?

File: content/docs/instrumenting/writing_exporters.md [hunk 4]
```diff
@@ -206,6 +206,22 @@ If your monitoring exposes a total like this, drop the total. If you have to
...
+time series. Additional information around a metric can be added via a info
```
Comment (jamtur01) [inline content/docs/instrumenting/writing_exporters.md]: `an info`

File: content/docs/instrumenting/writing_exporters.md (hunk 2 above)
Comment (brian-brazil) [reply content/docs/instrumenting/writing_exporters.md]: Changed

File: content/docs/instrumenting/writing_exporters.md (hunk 3 above)
Comment (brian-brazil) [reply content/docs/instrumenting/writing_exporters.md]: That reads okay to me.

File: content/docs/instrumenting/writing_exporters.md (hunk 4 above)
Comment (brian-brazil) [reply content/docs/instrumenting/writing_exporters.md]: Fixed

**Pull request merged**